src/providers/ObservationProvider.js

```diff
@@ -36,6 +36,13 @@ const ObservationProvider = ( { children }: Props ): Node => {
 
       // Live queries and objects emit notifications when something has changed that we can listen for.
       subscriptionRef.current = localObservations;
+      localObservations.addListener( ( arr, changes ) => {
+        // changes object has properties including insertions, modifications, and deletions
+        // so we can decide when we need obslist to rerender here. otherwise, it will listen for all changes
+        if ( changes.insertions.length > 0 || changes.modifications.length > 0 ) {
```

Comment:
Don't we *want* to listen to all changes to this collection, including deletions? Like if you delete an observation locally, shouldn't that be reflected in what you see on My Observations?

Reply:
Yep, fixed